### PR TITLE
[EC-169] Changed  Validator functions return type

### PR DESCRIPTION
**Pull request by KonradStaniec**

### CODE REVIEW

**Comment by AlanVerbner** [comment]
I think we also need to address this file, right? https://github.com/input-output-hk/mantis/blob/phase/daedalus/src/main/scala/io/iohk/ethereum/validators/BlockValidator.scala#L14

**Comment by ntallar** [comment]
Maybe we should also address this file as well?

- https://github.com/input-output-hk/mantis/blob/refactor/validator_return_type/src/main/scala/io/iohk/ethereum/validators/BlockHeaderValidator.scala#L9

File: src/test/scala/io/iohk/ethereum/validators/BlockValidatorSpec.scala [hunk 1]
```diff
@@ -46,12 +46,6 @@ class BlockValidatorSpec extends FlatSpec with Matchers {
     }
   }
 
-  "Block" should "return a block when verifing if a block body corresponds to a block header" in {
```
Comment (ntallar) [inline src/test/scala/io/iohk/ethereum/validators/BlockValidatorSpec.scala]: Minor detail: Why was this test removed?

Comment (KonradStaniec) [reply src/test/scala/io/iohk/ethereum/validators/BlockValidatorSpec.scala]: We are no longer returning `block` from `validateHeaderAndBody` and test was explicitly testing this. A valid case of a validator is covered by other tests.

**Pull request merged**